spec/models/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  context 'Tests For #Comment Model' do
+    before :each do
+      @user = User.create(name: 'King Cake',
+                          photo: 'https://media.istockphoto.com/id/650966412/photo/homemade-colorful-mardi-gras-king-cake.jpg?s=612x612&w=0&k=20&c=l2xfuI8SDGwDWOcbPvZz7wcCSMd0-IXOv03BJk-zSys=', bio: 'It is the ultimate the most deliciouse the jaguar King Cake.')
+
+      @post = Post.create(author: @user, title: 'Hello', text: 'This is my first post')
+
+      @comment = Comment.create(post: @post, author: @user, text: 'Hi Tom!')
+    end
+
+    it 'check if comment counter is incremented' do
+      expect(@post.comments.count).to eq 0
```

Comment:
- [ ] Well done writing your validations tests so far. However, you're missing a small thing to do for the tests to be valid and working correctly! This test should be expected to equal 1, not 0! Please use `describe` as required in this milestone. See the example down below!
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

Reply:
Hi @MahmoudBakr23 , thanks for the reminder it seems like there were a lot of errors that are stacked up after you solve this 😆 and after a long debuging session It's solved. 👍🏽